ydb/docs/en/core/cluster/short-access-control-notation.md

```diff
@@ -0,0 +1,67 @@
+# Short access control notation
+When describing or logging the permissions granted to users (e.g., in the recording of [audit logs](./audit-log.md)), a special short notation for access control may be used. The notation can vary slightly depending on the list of permissions and their inheritance from child objects.
+
+## Notation format
+
+Each entry begins with a `+` sign and consists of 2 or 3 attributes listed through the `:` symbol.
+These attributes are:
+- list of permissions. Mandatory.
+- SID of the subject granted permissions. Mandatory.
+- type of inheritance. Optional.
+
+#### Examples
+`+R:subject:O`
+`+W:subject`
+`+(SR|UR):subject`
+`+(SR|ConnDB):subject:OC+`
+
+## List of permissions {#access-rights}
+
+A short abbreviation is used to record each permission.
+
+#### Permission Groups
+Permission groups are unions of several permissions. Where possible, one of the groups will be indicated in the short notation.
+For example, `+R:subject` — permission to read.
+| Group	| Description |
+|:----:|:----|
+| `L` | (list) enumeration. Consists of permissions to read ACL attributes and describe objects.
```

Comment:
Does this include listing folder contents?

Reply:
DS (describe schema) afftects listing folder contents